references/obj_detection/utils.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+from operator import itemgetter
+from typing import Dict, List
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from doctr.utils.metrics import LocalizationConfusion
+
+__all__ = ["val_metric", ]
+
+
+def val_metric(targets: List[Dict], model: nn.Module, x: List[torch.Tensor]):
```

Comment:
Let's do a proper `doctr.utils.metrics`, I'll open a PR about this!

Reply:
Alright.